extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -192,7 +195,7 @@ describes.realWin('amp-bind', {
   });
 
   it('should verify string attribute bindings in dev mode', () => {
-    env.sandbox.stub(window, 'AMP_MODE', {development: true});
+    env.sandbox.stub(getMode(), 'development', true);
```

Comment:
It's used in a lot of test code. I'd consider it "[well known](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol#Well-known_symbols)" at this point.

Not to mention stub is for methods, not property values.